Examples/ExampleNaturalLanguageUnderstandingV1.cs

```diff
@@ -0,0 +1,133 @@
+﻿/**
+* Copyright 2019 IBM Corp. All Rights Reserved.
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+*      http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*
+*/
+
+using IBM.Watson.NaturalLanguageUnderstanding.V1;
+using IBM.Watson.NaturalLanguageUnderstanding.V1.Model;
+using IBM.Cloud.SDK.Utilities;
+using IBM.Cloud.SDK.Authentication;
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using IBM.Cloud.SDK;
+
+
+namespace IBM.Watson.Examples
+{
+    public class ExampleNaturalLanguageUnderstandingV1 : MonoBehaviour
+    {
+        #region PLEASE SET THESE VARIABLES IN THE INSPECTOR
+        [Space(10)]
+        [Tooltip("The IAM apikey.")]
+        [SerializeField]
+        private string iamApikey;
+		[Tooltip("The service URL (optional). This defaults to \"https://gateway.watsonplatform.net/natural-language-understanding/api\"")]
+        [SerializeField]
+        private string serviceUrl;
+        [Tooltip("The version date with which you would like to use the service in the form YYYY-MM-DD.")]
+        [SerializeField]
+        private string versionDate;
+        #endregion
+
+        private NaturalLanguageUnderstandingService service;
+        private string nluText = "IBM is an American multinational technology company headquartered in Armonk, New York, United States with operations in over 170 countries.";
+        private void Start()
+        {
+            LogSystem.InstallDefaultReactors();
+            Runnable.Run(CreateService());
```

Comment:
Can we do a check for `iamApikey` here and tell them to populate in the inspector?

Alternatively we can check if its null, then we can check if we can get it via env and if neither of those credentials are there we can throw an error telling them to add the apikey in the inspector.

Reply:
Adding the check in `createService`.